src/main/java/lotto/ui/Input.java

```diff
@@ -0,0 +1,57 @@
+package lotto.ui;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Input {
+
+    private static final String DELIMITER = ", ";
```

Comment:
6,5,4,3,2,1로 입력하면 에러가 발생합니다. parseInt에서 공백은 처리가 가능한 걸로 알고 있는데, DELIMITER = ","로 바꿔보면 어떨까요?
```suggestion
    private static final String DELIMITER = ",";
```